src/starlingbuilder/engine/tween/ITweenBuilder.as

```diff
@@ -44,16 +44,18 @@ package starlingbuilder.engine.tween
          * @param root of the DisplayObjects need to tween
          * @param paramsDict A dictionary of the mapping of every UIElement to its layout data
          * @param names array of names (e.g. ["container.button1", "container.button2") to be start tween, if null then start all the available tween
+         * @param trigger string used to run some of the tweens on an object, if null then start all available
          */
-        function start(root:DisplayObject, paramsDict:Dictionary, names:Array = null):void;
+        function start(root:DisplayObject, paramsDict:Dictionary, names:Array = null, trigger:String = null):void;
 
         /**
          * Stop tween according tweenData in layout
          *
          * @param root of the DisplayObject needs to be tween
          * @param paramsDict A dictionary of the mapping of every UIElement to its layout data, if null then stop all the available tweens
          * @param names array of names (e.g. ["container.button1", "container.button2") to be stop tween, if null then stop all the available tweens
+         * @param trigger string used to run some of the tweens on an object, if null then start all available
          */
-        function stop(root:DisplayObject, paramsDict:Dictionary = null, names:Array = null):void;
+        function stop(root:DisplayObject, paramsDict:Dictionary = null, names:Array = null, trigger:String = null):void;
```

Comment:
We should remove the trigger param, is not used.

Reply:
removed